modules/custom/openy_repeat/js/filters.js

```diff
@@ -0,0 +1,21 @@
+(function ($, Drupal) {
+
+  'use strict';
+
+  Drupal.behaviors.repeat_filters = {
```

Comment:
what does this javascript do? I would expect we implement logic related to searches within Vue app rather than separate javascript file. Lets discuss.

Reply:
Opted to move this piece over to it's own library so we could use it as a dependency for both the schedules and program search pages. I agree once we amalgamate the 2 apps this would make sense to go into the 1 application, that being said... do we have an ETA on if/when we would merge the too. If we don't go the library way we will have duplicate code. This would be a temporary, best practice piece of code. Thoughts?